docs/13-configuration.md

```diff
@@ -1,113 +1,128 @@
-
-Note that all of the settings in ssh.yaml will be overrided by parameters passed on the command line.
-
-## Spark Configuration
-
-The repository comes with default Spark configuration files which provision your Spark cluster just the same as you would locally. After running `aztk spark init` to initialize your working environment, you can view and edit these files at `.aztk/spark-defaults.conf`, `.aztk/spark-env.sh` and `.aztk/core-site.xml`. Please note that you can bring your own Spark configuration files by copying your `spark-defaults.conf`, `spark-env.sh` and `core-site.xml` into your `.aztk/` direcotry.
-
-The following settings available in `spark-defaults.conf` and `spark-env.sh` are not supported:
-
-`spark-env.sh`:
-- SPARK\_LOCAL\_IP
-- SPARK\_PUBLIC\_DNS
-- SPARK\_MASTER\_HOST
-- SPARK\_MASTER\_PORT
-- SPARK\_WORKER\_PORT
-- SPARK\_MASTER\_WEBUI\_PORT
-- Any options related to YARN client mode or Mesos
-
-`spark-defaults.conf`:
-- spark.master
-
-Also note that this toolkit pre-loads wasb jars, so loading them elsewhere is not necessary.
-
-## Configuring Spark Storage
-
-The Spark cluster can be configured to use different cloud supported storage offerrings (such as Azure Storage Blobs, Azure Data Lake Storage, or any other supported Spark file system). More information can be found in the [Cloud Storage](./30-cloud-storage.md) documentation.
-
-## Placing JARS
-
-Additional JAR files can be added to the cluster by simply adding them to the *.aztk/jars* directory. These JARS will automatically be added to Spark's default JAR directory. In the case of a naming conflict, the file in *.aztk/jars* will **overwrite** the file in the cluster. Typically new JARS must be registered with Spark. To do this, either run the Spark Submit command with a path to the JARS
-
-```sh
-aztk spark cluster submit --id <my_cluster_id> --jars $SPARK_HOME/jars/my_jar_file_1.jar <my_application> <my_parameters>
-```
-
-Or update the *.aztk/spark-default.conf* file as shown below to have it registered for all Spark applications.
-
-```sh
-spark.jars $spark_home/jars/my_jar_file_1.jar,$spark_home/jars/my_jar_file_2.jar
-````
-
-Note: _This tool automatically registers several JARS for default cloud storage in the spark-default.conf file. If you want to modify this file, simply append any additional JARS to the end of this list_.
+# Configuration Files
+This section refers to the files in the directory *.aztk* that are generated from the `aztk spark init` command.
+
+## *cluster.yaml*
+
+The core settings for a cluster are configured in the *cluster.yaml* file. Once you have set your desired values in *.aztk/cluster.yaml*, you can create a cluster using `aztk spark cluster create`.
+
+This is the default cluster configuration:
+
+```yaml
+# id: <id of the cluster to be created>
+id: spark_cluster
+
+# vm_size: <vm-size, see available options here: https://azure.microsoft.com/en-us/pricing/details/virtual-machines/linux/>

```

Comment:
probably worth updating the link to batch pricing:
https://azure.microsoft.com/pricing/details/batch/